app/views/dishes/_form.html.haml

```diff
@@ -1,11 +1,25 @@
--# frozen_string_literal: true
-= simple_form_for(@dish) do |f|
-  = f.error_notification
-  = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
-
-  .form-inputs
-    = f.input :name
-    = f.input :price
-
-  .form-actions
-    = f.button :submit
+%form
+  .row.mb-3
+    %label.col-sm-2.col-form-label{:for => "@Name"} Name
```

Comment:
What does for mean here? Use correct hash syntax
%label.col-sm-2.col-form-label {for: "@Name"}= "Name"